xml/user-glance.xml

```diff
@@ -835,7 +835,7 @@ managed at a namespace level.
                     mapping in the following format:</para>
             <screen>{'images': [
   {'uri': 'http://glance.openstack.example.org/v1/images/71c675ab-d94f-49cd-a114-e12490b328d9',
-   'name': 'Ubuntu 10.04 Plain',
+   'name': 'SLE 15',
```

Comment:
Not positive but it might be better to call it SLES 15 or SUSE Linux Enterprise Server 15

Reply:
In our engineering cloud we call it SLES15

Reply:
Ah - I shouldn't have been so literal. Sorry.

Reply:
@dmpop do we have official style guide naming conventions we need to stick to here?

Reply:
We have entities for both SLE and SUSE Linux Enterprise Server. But I'm not sure whether it makes sense to use them here. Just use SLES 15.